packages/@okta/vuepress-site/docs/guides/build-sso-integration/before-you-begin/openidconnect/prep.md

```diff
@@ -9,9 +15,10 @@ Before you create a new OIDC integration in Okta:
    * A single page app (SPA) is a web application that is contained on a single web page. All code is retrieved when the page is loaded initially - the page doesn't reload or refresh. A SPA application cannot keep running on a server.
 -->
 
+1. Have your application developed and tested - with a front-end (for example, JavaScript and HTML) and back-end (for example, middleware and database software) stack, with services available through APIs and accepting HTTP connections.
```

Comment:
Have your application developed and tested with a front-end (for example, JavaScript and HTML) and back-end (for example, middleware and database software) stack, with services available through APIs, and accepting HTTP connections.

Reply:
Have your application developed and tested, with both a front-end (for example, JavaScript and HTML) and a back-end (for example, middleware and databases) software stack, along with services available through APIs, and accepting HTTP connections.